client/landing/gutenboarding/onboarding-block/index.ts

```diff
@@ -13,9 +13,9 @@ import edit from './edit';
 export const name = 'automattic/onboarding';
 
 export const settings: BlockConfiguration< Attributes > = {
-	title: NO__( 'Onboarding' ),
+	title: __( 'Onboarding' ),
```

Comment:
This doesn't need translating, unless it pops up for screen-readers somewhere and then we should have something smarter here. Maybe just "WordPress.com" if it cannot be left out?

Reply:
144277448cf452b8feb4339166d2b703313b05c6